packages/@uppy/dashboard/src/Dashboard.tsx

```diff
@@ -145,7 +151,7 @@ interface DashboardMiscOptions<M extends Meta, B extends Body>
   hideRetryButton?: boolean
   hideUploadButton?: boolean
   metaFields?: MetaField[] | ((file: UppyFile<M, B>) => MetaField[])
-  nativeCameraFacingMode?: ConstrainDOMString
+  nativeCameraFacingMode?: 'user' | 'environment' | ''
```

Comment:
Why this change?

Reply:
Because these are the only possible values https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/capture.